README.md

```diff
@@ -1,2 +1,58 @@
-# knex-elevator
-Control a mutli-shaft elevator built from K'Nex.
+# KNEX Elevator
```

Comment:
This file needs plenty of work, but I figure we can start filling this in as we get the system stood up more.